tony-core/src/main/java/com/linkedin/tony/Constants.java

```diff
@@ -13,6 +13,10 @@
   // File Permission
   public static final FsPermission perm770 = new FsPermission((short) 0770);
 
+  // History Server related constants
+  public static final String JOBS_SUFFIX = "jobs";
+  public static final String CONFIG_SUFFIX = "config";
```

Comment:
I would just call these constants `JOBS` and `CONFIG` and remove the `_SUFFIX` part.

Feel free to address in a separate PR or as part of another PR.